<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="Dimensions" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>30</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>2</number>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Dimensions</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MakeBounds" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Make Bounds</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCx" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCy" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCx" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCy" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCz" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCz" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>180</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>180</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCz</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpanThreshold" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
   <property name="value" >
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>   Minimum Level</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaximunLevel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>60</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>2</number>
   </property>
   <property name="maximum" >
    <number>11</number>
   </property>
   <property name="value" >
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>   Maximum Level</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinimumLevel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>80</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="maximum" >
    <number>10</number>
   </property>
   <property name="value" >
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
